index.js

```diff
@@ -68,7 +88,20 @@ function checkIfAnyMovieHasRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+  // if(movies.imdbID !== id){
+  //   return null;
+  // }
+  if(!movies.length){
+    throw "Empty";
+  }
+  
+let movIdArr = movies.find((obj)=>{
+  return obj.imdbID === id;
+});
+
+  return movIdArr;
```

Comment:
.find returns undefined if it can't find a match. But the tests say you have to return null if that's the case, not undefined. You needed to use a conditional to check if you got a successful match, or undefined.
```suggestion
  if(movIdArr === undefined) {
  return null
  } else {
  return movIdArr
  }
```
You were really close to 100% passing this one, nice job Trusty!